### DISCUSSION TITLE
Repository `Type` filters are problematic

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#14162] [Other Feature Feedback, Questions, & Ideas]
Consider:
https://github.com/docs
<img width="852" alt="image" src="https://user-images.githubusercontent.com/2119212/161416231-75dd3e67-6133-406e-823d-92e959deace7.png">

I'm kinda curious as to why an org would pin a whole bunch of archived repositories, but hey...

Ok, so the top line says Repositories `(8)`

https://github.com/orgs/docs/repositories

<img width="753" alt="image" src="https://user-images.githubusercontent.com/2119212/161416286-52740bbe-74b8-42e7-9641-0af072f7e78b.png">

I'm looking for the non-archived repositories.

I suppose I could try to visually figure out which repositories aren't archived. But at this hour, I'd fail miserably. (I tried a couple of times and got it wrong each time. At one point, I guessed there were 4, and there kinda are. Sources says 3.)

There's a `Type` dropdown:
<img width="235" alt="image" src="https://user-images.githubusercontent.com/2119212/161416303-4453eda4-a9ec-4eba-8c7b-ab3a5e40411f.png">

So, I can filter for `Archived`. But I can't find any way to filter for _not_ `Archived`. (8 - 4 = 4, so this is the set I'm looking for. `Sources` will yield a different answer of `3`, which is interesting, but...)

I'm looking for the opposite of this:
https://github.com/orgs/docs/repositories?q=&type=archived&language=&sort=
<img width="756" alt="image" src="https://user-images.githubusercontent.com/2119212/161416343-9306bdda-9dd4-438c-b100-2f4e22d32363.png">

---

Really, I'm looking for the ability to find `not archived`, `not mirrors`, `not forks`, and maybe `not template`.

It turns out that `sources` might be the answer.

But, here's the thing:

https://docs.github.com/en/repositories/archiving-a-github-repository/archiving-repositories

Says that an archived repository can still be a source for another repository. Which means that `Sources` doesn't quite mean what it thinks it means.

I have no idea what it means. And I will file a bug against for it to be documented, as afaict this widget isn't documented.

**Comment by tommy-gilligan** [comment]
Love that there is filtering but not being able to filter out archived repos is a problem here also.

**Reply by Marychoffel22** [reply]
Assigned

**Comment by Marychoffel22** [comment]
Pull requests

**Comment by Marychoffel22** [comment]
Assigned

**Comment by Marychoffel22** [comment]
Full access

**Comment by Marychoffel22** [comment]
iCloud@smtp.com
